### DISCUSSION TITLE
localhost connection issues.

### DISCUSSION THREAD

**Discussion by Mukioyu** [ppy/osu#16300] [Q&A] [answered]
Hello,
I was downloading the osu!lazer source code and I modified it a bit because I wanted to test out what I can modify. I did `dotnet run --project osu.Desktop` and it worked but if I want to login on my localhost server or any other server it just says "Your IP address is locked. Please wait a few minutes." I've been waiting for a while now and on osu!stable it works perfectly fine. I only modified DiscordRichPresence.cs and VersionManager.cs
Thanks in advance,
Muki.

**Comment by peppy** [comment, marked as answer]
You probably locked your IP on the development server. Wait an hour or so and it will resolve. There is no "localhost" server.
